inc/taxonomies.php

```diff
@@ -443,29 +474,3 @@ function largo_first_headline_in_post_array($array) {
 
 	return $headline;
 }
-
```

Comment:
`unregister_post_types_taxonomy`'s check was moved into general taxonomy-registration function in `inc/taxonomies.php`. It was moved there for consistency with other taxonomy-registration functions, but in any case it needed to be integrated with the function that registered the `post-types` taxonomy, instead of registering a taxonomy without any labels.

I'm not sure why `unregister_series_taxonomy` was removed; but the same registration fix should be applied to the `series` taxonomy.